cardinal/ecs/world.go

```diff
@@ -486,7 +494,9 @@ func (w *World) Tick(ctx context.Context) error {
 	}
 	w.tick++
 	w.receiptHistory.NextTick()
-
+	w.logger.Info().
+		Str("tick", tickAsString).
+		Msg("Tick ended")
```

Comment:
Can we also log how long it took for the tick to finish executing?

Reply:
This is done.